regression/contracts-dfcc/invar_havoc_dynamic_array_const_idx/main.c

```diff
@@ -10,7 +10,7 @@ void main()
   data[4] = 0;
 
   for(unsigned i = 0; i < SIZE; i++)
-    __CPROVER_loop_invariant(i <= SIZE)
+    __CPROVER_assigns(data[1], i) __CPROVER_loop_invariant(i <= SIZE)
```

Comment:
Is this change now strictly necessary given the additional widening?

Reply:
Yes, new addition widening will infer `object_whole(data), i` which is not accurate for this test.